src/runmode-napatech.c

```diff
@@ -194,14 +194,6 @@ static void *NapatechConfigParser(const char *device)
         return NULL;
     }
 
-    /* Set the host buffer allowance for this stream
-     * Right now we just look at the global default - there is no per-stream hba configuration
-     */
-    if (ConfGetInt("napatech.hba", &conf->hba) == 0) {
-        conf->hba = -1;
-    } else {
-        SCLogWarning("Napatech Host Buffer Allocation (hba) will be deprecated in Suricata v7.0.");
```

Comment:
I think it would be good to keep a warning if `hba` is set in the yaml, e.g.:

"Napatech Host Buffer Allocation (hba) support has been removed, see ticket #XXXX"

We need that ticket in any case too.